README.md

```diff
@@ -199,6 +199,23 @@ Building this provider requires a 64 bit Linux or MacOS build system with the fo
 2. Run `./gradlew release`
 3. The resulting jar is in `build/lib`
 
+### Repackaging ACCP into Uber/Fat Jars
```

Comment:
I think you meant for this to either be header level 4 or 5. It definitely doesn't make sense for FIPS builds to be a subsection of this repackaging section.